core/src/main/java/org/mule/api/security/tls/RestrictedSSLSocketFactory.java

```diff
@@ -100,4 +102,24 @@ private SSLSocket restrictCipherSuites(SSLSocket socket)
         socket.setEnabledProtocols(enabledProtocols);
         return socket;
     }
+
+    public static synchronized SocketFactory getDefault()
+    {
+        if(defaultSocketFactory != null)
+        {
+            return defaultSocketFactory;
+        }
+
+        try
+        {
+            TlsConfiguration configuration = new TlsConfiguration(null);
+            configuration.initialise(true, null);
+            defaultSocketFactory = new RestrictedSSLSocketFactory(configuration.getSslContext(), configuration.getEnabledCipherSuites(), configuration.getEnabledProtocols());
+        }
+        catch (Exception e)
+        {
+            throw new RuntimeException("Could not create the default RestrictedSSLSocketFactory", e);
```

Comment:
Throw a MuleRuntimeException just so it's clear we are failing and not something else.